From aacd41cfd4974d9cfbae8467e2d8985f42e203d6 Mon Sep 17 00:00:00 2001
From: Julie Jeongeun Kim <jkim@igalia.com>
Date: Wed, 17 Oct 2018 18:03:55 +0900
Subject: [PATCH] Add RestartSec

---
 files/launch/WebAppMgr.service | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/files/launch/WebAppMgr.service b/files/launch/WebAppMgr.service
index 5a5fe0a..942eaa8 100644
--- a/files/launch/WebAppMgr.service
+++ b/files/launch/WebAppMgr.service
@@ -12,8 +12,8 @@
 
 [Unit]
 Description="WebAppMgr is responsible for running web apps and manage their lifecycle"
+Requires=weston.service
 After=weston.service
-Wants=weston.service
 
 [Service]
 Type=simple
@@ -21,6 +21,7 @@ OOMScoreAdjust=-1000
 EnvironmentFile=-/etc/default/WebAppMgr.env
 ExecStart=/usr/bin/WebAppMgr --no-sandbox --remote-debugging-port=9998 --user-data-dir="/home/0/wamdata" --webos-wam
 Restart=on-failure
+RestartSec=50
 
 [Install]
 WantedBy=multi-user.target
-- 
2.14.1

